README.md

```diff
@@ -45,8 +45,9 @@ public class SomeActivity extends Activity implements BillingProcessor.IBillingH
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
-    bp = new BillingProcessor(this, "YOUR LICENSE KEY FROM GOOGLE PLAY CONSOLE HERE", this);
-    bp.initialize();
+    	bp = new BillingProcessor(this,LICENSE_KEY,MERCHANT_ID,this);
+        bp.connect(this); //Connection required based on google Version 4 for inapp lib
```

Comment:
can this be moved into existing `initialize` method?

Reply:
Yes that is possible, I have tried to combine them but I got some errors which i have not fixed yet. However when i get them resolved I will pull a new request. Another quick fix is to add the _connect(this)_ method into the _initialize_ method. or is that your suggestion?